client/components/Register.jsx

```diff
@@ -49,27 +58,55 @@ class Register extends React.Component {
         <br/>
         <label htmlFor='surname'>Surname</label>
         <br/>
-        <input type='text' placeholder='Last name' name='surname'
+        <input 
+          type='text' 
+          placeholder='Last name' 
+          name='surname'
           className='u-full-width'
-          value={this.state.name}
+          value={this.state.surname}
           onChange={this.handleChange}
         />
         <br/>
         <br/>
         <label htmlFor='hash'>Hash</label>
         <br/>
-        <input type='password' placeholder='Password' name='password'
+        <input 
+          type='password' 
+          placeholder='Password' 
+          name='password'
           className='u-full-width'
-          value={this.state.description}
+          value={this.state.password}
           onChange={this.handleChange}
         />
         <br/>
         <br/>
-        <label htmlFor='role'>Role</label>
+        <br/>
+        <button 
+          type='submit' 
+          className='button-primary' 
+          value='Add' 
+          onClick={this.registering}
+          >
+        Register</button>
+      </form>
+    )
+  }
+}
+
+const mapStateToProps = ({auth}) => {
+  return {
+    auth
+  }
+}
```

Comment:
If you want to refactor lines 96-100, you can do it with one line: `const mapStateToProps = ({auth}) => ({auth})` :smile_cat: